mapfishapp/src/deb/resources/etc/georchestra/mapfishapp/addons/quicksearch/config.json

```diff
@@ -0,0 +1,52 @@
+[{
+        "id": "quicksearch_0",   // unique & stable string identifier for this addon instance
```

Comment:
comments in this file should be mentioned in the README (if not already present)